src/easier68k/core/util/find_module.py

```diff
@@ -0,0 +1,30 @@
+# This *is* actually a necessary import due to using "reflection" style code further down
+# noinspection PyUnresolvedReferences
+from ..opcodes import *
+from types import ModuleType
+import sys
+
+valid_opcodes_module = [
+    'easier68k.core.opcodes.move'
+]
+
+valid_opcodes = [
+    x.split('.')[-1].upper() for x in valid_opcodes_module
+]
+
+def find_module(opcode: str) -> (ModuleType, type): # classes are of type "type" Really python?
```

Comment:
maybe we should have it return type `OpCode` (the base class)

Reply:
Well, it returns the class itself. Saying it returns OpCode would mean it returns an instance.

Reply:
oh, so `type` should be accurate then